Packs/Office365AndAzureAuditLog/Integrations/MicrosoftPolicyAndComplianceAuditLog/README.md

```diff
@@ -100,84 +85,3 @@ Use the o365-search-auditlog command to search the unified audit log. This log c
 | O365AuditLog.UserType | Number | The type of user who performed the operation. | 
 | O365AuditLog.Version | Number | The version of the log. | 
 | O365AuditLog.Workload | String | The Office 365 service where the activity occurred. | 
-
-
-#### Command Example
-```!o365-auditlog-search start_date="01/01/21" end_date="01/02/21" result_size=1```
-
-#### Context Example
```

Comment:
Let's leave the example since it will help users better understand.

Reply:
Sure, i havent finished with the docs just yet, it was decided to stay on V1 and make it clear to the users that their instances will have to be reconfigured.